homeassistant/components/edl21/sensor.py

```diff
@@ -283,7 +290,7 @@ async def async_setup_platform(
         hass,
         DOMAIN,
         "deprecated_yaml",
-        breaks_in_ha_version="2023.2.0",
+        breaks_in_ha_version="2023.5.0",
```

Comment:
Please try to avoid mixing changes: this invalid value could be a standalone PR (I know it's very small, but since this PR needs changes already I suggest you move it out).